dpnp/backend/kernels/dpnp_krnl_fft.cpp

```diff
@@ -187,9 +186,19 @@ void dpnp_fft_fft_mathlib_compute_c(const void* array1_in,
     desc.set_value(mkl_dft::config_param::PLACEMENT, DFTI_NOT_INPLACE);
     desc.commit(DPNP_QUEUE);
 
-    event = mkl_dft::compute_forward(desc, array_1, result);
+    const size_t n_iter =
+        std::accumulate(input_shape, input_shape + shape_size - 1, 1, std::multiplies<shape_elem_type>());
 
-    event.wait();
+    const size_t shift = input_shape[shape_size - 1];
```

Comment:
Please keep in mind that input shape is able to be empty.

Reply:
I would check `!shape_size` in function `dpnp_fft_fft_mathlib_compute_c` instead of `!input_shape` in function `dpnp_fft_fft_mathlib_c`.
Imagine that you want to call `dpnp_fft_fft_mathlib_compute_c` with empty shape as input in other place.